test/utils/infer.py

```diff
@@ -16,9 +16,10 @@
 from typing import Any, Tuple
 
 import test.utils.helper as helper
-
 import tico.utils
 import tico.utils.model
+import torch
+from tico.interpreter.infer import flatten_and_convert
```

Comment:
How about order like this?

1. Standard library
2. Third library
3. Own module

```python
import os
from typing import Any, Tuple

import torch
import tico.utils
import tico.utils.model
from tico.interpreter.infer import flatten_and_convert

import test.utils.helper as helper
```